docs/source/settingup.rst

```diff
@@ -1,29 +1,90 @@
 .. include:: ./links.inc
 
-Setting up
+.. _install:
+
+Quickstart
 ==========
 
 Requirements
+^^^^^^^^^^^^
+
+DataLad and ``datalad-osf`` are available for all major operating systems (Linux, MacOS, Windows 10).
+
+   An OSF_ account
+       You need an OSF account to be able to interact with it. If you don't have an account yet, `register here <https://osf.io/register>`_ -- its free!
+
+   DataLad
+       If you don't have DataLad_ and its underlying tools (`git`_, `git-annex`_) installed yet, please follow the instructions from `the datalad handbook <http://handbook.datalad.org/en/latest/intro/installation.html>`_.
+
+   [optional] An account on a Git repository hosting site
+       You should consider having an account on one or more repository hosting sites such as `GitHub <https://github.com/join>`__ , `GitLab <https://gitlab.com/users/sign_up>`_, `Bitbucket <https://bitbucket.org/account/signup/>`_ or similar.
+
+Installation
 ------------
 
-- DataLad
+``datalad-osf`` is a Python package available on `pypi <https://pypi.org/project/datalad-osf/>`_ and installable via pip_.
 
-Before being able to use the extension, you need to have DataLad installed, which relies on `git-annex`_, `git`_ and `Python`_.
-If you don't have DataLad installed yet, please follow the instructions from `the datalad handbook <http://handbook.datalad.org/en/latest/intro/installation.html>`_.
+.. code-block:: bash
 
-- An account on the OSF
+   # create and enter a new virtual environment (optional)
+   $ virtualenv --python=python3 ~/env/dl-osf
+   $ . ~/env/dl-osf/bin/activate
+   # install from PyPi
+   $ pip install datalad-osf.
 
-You need an OSF account to be able to interact with it. If you don't have an account yet, `register here <https://osf.io/register>`_.
+If you want to use the most recent development version, clone its `GitHub repository <https://github.com/datalad/datalad-osf>`_ to your local computer and install the package in editable mode:
 
-- An account on a git repository hosting site
+.. code-block:: bash
 
-You should consider having an account on one or more repository hosting sites such as `GitHub <https://github.com/join>`__ , `GitLab <https://gitlab.com/users/sign_up>`_, `Bitbucket <https://bitbucket.org/account/signup/>`_ or similar"
+   # create and enter a new virtual environment (optional)
+   $ virtualenv --python=python3 ~/env/dl-osf
+   $ . ~/env/dl-osf/bin/activate
+   # install from local repository clone
+   $ pip install -r requirements-devel.txt
+   $ pip install -e .
```

Comment:
I'd say we can skip this part. I'd assume that anyone willing and capable of hacking on a python package would know their way around. But I am also OK with keeping it in.

Reply:
done